config.json

```diff
@@ -61,6 +61,13 @@
       ]
     },
     {
+      "slug": "acronym",
+      "difficulty": 2,
+      "topics": [
+        "String Manipulation"
```

Comment:
I noticed https://github.com/exercism/xhaskell/issues/274#issuecomment-253597062 mentions `String` instead of `String Manipulation`, I'm ok with either term.

The only problem I see with this topic affecting ordering decision is the breadth of difficulty that it might include.

For instance, taken from https://github.com/exercism/xhaskell/issues/274#issuecomment-253597062, we know `Maybe`, `ADT`, `Either` and `IO Monads` require enough understanding so that it can be used for ordering the exercises. But take a few `String` related exercises:

* `Bob`: **1**
* `Pangram`: **1**
* `Run Length Encoding`: **1**
* `Acronym`: **2**
* `Beer song`: **3**
* `Anagram`: **4**
* `Ocr Numbers`: **5**
* `Pig Latin`: **6**

They are all related to _String Manipulation_, but don't have any other subtopic (except for `Beer Song`, which includes `Refactoring`, although I'm not sure if I agree). Still, they have fairly different levels of difficulty. Perhaps there are more underlying topics that we haven't discovered yet, but currently they can't rely only on `String` to be ordered.

That said, do we have a clear/fixed way to order the exercises based on their topics? For example:

* Sort by most difficult topic (likely the current approach); or
* Assign a weight to each topic and sort the exercises by the sum of their weight.